capstone/src/main/webapp/index.html

```diff
@@ -5,17 +5,80 @@
     <title>Studentia-Homepage</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <script type = "text/javascript" src = "https://code.jquery.com/jquery-2.1.1.min.js"></script>             
+    <script src = "https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.3/js/materialize.min.js"></script>   
     <!--Import Google Icon Font-->
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
     <!--Import materialize.css-->
     <link type="text/css" rel="stylesheet" href="css/materialize.min.css"  media="screen,projection"/>
-
+    <link type="text/css" rel="stylesheet" href="css/materialize.css"  media="screen,projection"/>
     <!--Let browser know website is optimized for mobile-->
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
   </head>
   <body>
-     <h3>Hompage</h3>
+    <nav>
+      <div class="nav-wrapper">
+        <a href="#" class="brand-logo">Studentia</a>
+        <ul id="nav-mobile" class="right hide-on-med-and-down">
+          <li><a href="findStudySet.html">Search</a></li>
+        </ul>
+      </div>
+    </nav>
+    <section class="introduction">      
+      <div class="row"> 
+        <div class="col l4 m4 s12 push-l1">
+          <h1 class="white-text text-darken-4">Studentia</h1>
+          <sub class="flow-text"> Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut pharetra feugiat vehicula. Morbi vitae varius felis.
+                Ut quis interdum dui. Sed pulvinar eros non dictum molestie. Mauris ex arcu, dictum at justo ut, gravida maximus nibh.
+                Proin id mollis magna
+          </sub>
+        </div>
+        <div class="col l8 m8 s12 push-l1">
+           <h3>This will be a slideshow walkthrough</h3>
+        </div>
+      </div>
+    </section> 
+    <section class="tab-description">
+      <div class="row">
+        <div class="col s12 m12 l12">
+          <ul class="tabs">
+            <li class="tab col s4"><a href="#termsOfUsage">Terms of Usage</a></li>
+            <li class="tab col s4"><a href="#About" >About</a></li>
+            <li class="tab col s4"><a href="#getStarted">Get Started</a></li>
+          </ul>
+          <div class="col s12 tb" id="termsOfUsage">
```

Comment:
Change all IDs to lowercase, hyphen separated: https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters. Remember to update the anchor links above as well.